File: primerTrimestre/java basico/2alternativas/Ej3alt.java
/**Escribe un programa que lea un número e indique si es par o impar.
 * 
 * @author Antonio Quesada
 */


import java.util.Scanner;

public class Ej3alt {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos el número.
    int num;
    
    //Pedimos el dato número.
    System.out.println("Introduce un numero: ");
    num = s.nextInt();
    
    //Calculamos si es par si el número es divisble entre 2 y devuelve como resto 0.
    if (num%2 == 0) {
      System.out.println("El número es par");
    } else {  //Si no es par muestra que es impar.
      System.out.println("El número es impar");
    }

  }

}

File: primerTrimestre/introduccion_programacion_python/2alternativas_py/ej3alt.py
# Programa: ej3alt.py
# Propósito: Escribe un programa que lea un número e indique si es par o impar.
# Autor: Antonio Quesada
# Fecha: 23/10/2019.
#
# Variables a usar:
#   * num
#
# Algoritmo:
#   Leemos y pedimos el num..
#   Si el resto de la división del número entre 2 es igual a 0 es par.
#   Si no es impar.

# Petición de datos
num = float(input("Introduce un número: "))
print("------------------------------------------")

# Salida
if num%2 == 0:
    print("El número", num,"es par")
else:
     print("El número", num,"es impar")
